CHANGELOG.rst

```diff
@@ -15,6 +15,15 @@ Change Log
 
 Unreleased
 ----------
+
+[4.1.7]
```

Comment:
You'll also need to bump the package version to 4.1.7

Reply:
@kdmccormick I did bump the package number in enterprise/__init__.py, are there other places I'm not aware of?